examples/zookeeper/README.md

```diff
@@ -0,0 +1,63 @@
+Replicated Zookeeper Template
+=============================
+
+ZooKeeper is a distributed, open-source coordination service for distributed
+applications. It exposes a simple set of primitives that distributed
+applications can build upon to implement higher level services for
+synchronization, configuration maintenance, and groups and naming. It is
+designed to be easy to program to, and uses a data model styled after the
+familiar directory tree structure of file systems. It runs in Java and has
+bindings for both Java and C.
+
+For more informations about Zookeeper visit:
+
+http://zookeeper.apache.org/doc/r3.4.6/zookeeperOver.html
+
+### Pre-requires
+
+You can 'pre-pull' the Docker image used by this template by:
+
+```
+$ docker pull mfojtik/zookeeper-346-jdk7
```

Comment:
@dmcphers zookeeper-34-fedora20... but I already pushing the detailed
version....

On Wed, Apr 29, 2015 at 7:24 PM, Dan McPherson <notifications@github.com>
wrote:

> In examples/zookeeper/README.md
> <https://github.com/openshift/origin/pull/1965#discussion_r29358576>:
>
> > +applications can build upon to implement higher level services for
> > +synchronization, configuration maintenance, and groups and naming. It is
> > +designed to be easy to program to, and uses a data model styled after the
> > +familiar directory tree structure of file systems. It runs in Java and has
> > +bindings for both Java and C.
> > +
> > +For more informations about Zookeeper visit:
> > +
> > +http://zookeeper.apache.org/doc/r3.4.6/zookeeperOver.html
> > +
> > +### Pre-requires
> > +
> > +You can 'pre-pull' the Docker image used by this template by:
> > +
> > +```
> > +$ docker pull mfojtik/zookeeper-346-jdk7
>
> @mfojtik <https://github.com/mfojtik> can we be a little less specific on
> the version as well? I assume that's 3.4.6. Can we make it 3 or 3.4?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/1965/files#r29358576>.
>



-- 
-----------------------------------------------
Michal Fojtik <mi@mifo.sk <mfojtik@mifo.sk>>
Red Hat OpenShift, Engineering